optimum/intel/openvino/modeling_seq2seq.py

```diff
@@ -435,13 +412,13 @@ class OVEncoder:
             The OpenVINO inference request associated to the encoder.
     """
 
-    def __init__(self, model: openvino.runtime.Model, device: str, ov_config: Dict, main_input_name="input_ids"):
+    def __init__(self, model: openvino.runtime.Model, device: str, parent_model: OVModelForSeq2SeqLM):
         self.model = model
         self._device = device
+        self.parent_model = parent_model
```

Comment:
can be used to get the device also wdyt ?

Reply:
Yes, I was a bit hesitant to make more changes than needed, but should have done this :-) Thanks, I'll change it.

Reply:
perfect, thanks @helena-intel !